src/config/dynamic-theme-fixes.config

```
@@ -10156,6 +10156,30 @@ IGNORE INLINE STYLE
 
 ================================
 
+modmapper.com
+
+CSS
+button[class^="ToggleLayersControl_heatmap"] > span {
+    background-image: url(/img/heatmap.svg) !important;
+    filter: invert(1) !important;
+}
+button[class^="ToggleLayersControl_grid"] > span {
+    background-image: url(/img/grid.svg) !important;
+    filter: invert(1) !important;
+}
+button[class^="ToggleLayersControl_labels"] > span {
+    background-image: url(/img/labels.svg) !important;
+    filter: invert(1) !important;
+}
```

```suggestion
INVERT
button[class^="ToggleLayersControl"] > span
```

Should be good enough?

---

I'll have to test in the morning, but I do believe I tested that. IIRC, the problem was with Dark Reader doing weird stuff to the SVGs that I couldn't (easily) undo with CSS, given my current knowledge.

Though I do see how I could minify the file by using that selection you used to apply that invert filter, should the inversion not work.